pkg/context/context.go

```diff
@@ -19,6 +19,11 @@ type Context struct {
 	SnapshotPath         string
 	SnapshotRoot         string
 	SnapshotExclusive    bool
+	PathEtc              string
+	PathProc             string
+	PathRun              string
+	PathSys              string
+	PathVar              string
```

Comment:
These are all Linux-specific and should be in `pkg/linuxpath` package.

For the generic context package, you can do something like this in `pkg/option/option.go`:

```go
// PathOverrides is a map, keyed by the string name of a mount path, of override paths
type PathOverrides map[string]string

// WithPathOverrides supplies path-specific overrides for the context
func WithPathOverrides(overrides PathOverrides) *Option {
	return &Option{
        		PathOverrides: overrides,
       	}
}
```

Reply:
You are right. The only reason why I put them here is to (try to) avoid the circular dependency issue. I'll try out the approach you suggested, which is cleaner indeed.